multicast/__main__.py

```
@@ -574,12 +574,12 @@ class McastDispatch(mtool):
 	"""
 
-	__proc__ = """multicast"""
+	__proc__ = "multicast"
 
-	__prologue__ = """The Main Entrypoint."""
+	__prologue__ = "The Main Entrypoint."
 
 	__epilogue__ = str(
-		"""When called from the command line the __main__ component handles the CLI dispatch."""
+		"When called from the command line the __main__ component handles the CLI dispatch."
 	)
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)